bin.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+const glslifyBundle = require('glslify-bundle')
+const glslifyDeps   = require('glslify-deps')
+const glslResolve   = require('glsl-resolve')
+const minimist      = require('minimist')
+const path          = require('path')
+const bl            = require('bl')
+const fs            = require('fs')
+
+const INPUT  = path.join(__dirname, '__bogus__')
+const argv   = minimist(process.argv.slice(2))
+const depper = glslifyDeps({ readFile: readFile })
+
+var input = ''
+
+if (argv.help) return help()
+if (!argv._.length && process.stdin.isTTY) return help()
+
+argv.t = argv.t || []
+argv.t = Array.isArray(argv.t) ? argv.t : [argv.t]
+argv.t.forEach(function(tr) {
+  depper.transform(tr)
+})
+
+//
+// Build dependency tree, then output
+//
+if (argv._.length) {
+  return depper.add(argv._[0], output)
+}
+
+process.stdin.resume()
```

Comment:
I think the `.pipe` below *should* start flowing the `stdin` stream without `.resume()`.